### PR TITLE
Workspace setup

### PR DESCRIPTION
**Pull request by ftomassetti**
Following up on #9 this is an attempt to divide the APIs into modules that could potentially be activated separately.

For now I just put the `init` route into a separate package called `dbadmin`
I then moved all the rest of the code to a package called `core`.
Based on suggestions by @dslmeinte I made the tsconfig.json files of the packages inherit from the root tsconfig.json.

I then created some tasks in the root projects which basically invoke the same tasks in the core package or the core and the dbadmin packages.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -12,6 +12,9 @@
     "forceConsistentCasingInFileNames": true,
     "strict": false,
     "skipLibCheck": true,
-    "sourceRoot": "./src"
-  }
+    "sourceRoot": "./src",
+  },
+  "include": [
```
Comment (dslmeinte) [inline tsconfig.json:17]: This set up is a bit weird: the packages are in `packages/dbadmin/`, resp. `src/`, but this `tsconfig.json` just finds everything. I think it'd be better to have a `core` and `dbadmin` package, and use the same setup as e.g. in the `lionweb-typescript` repository: one top-level `tsconfig.json`, and one per package that reuses the top-level one, but has a specific `sourceRoot` spec.

File: package.json [hunk 2]
```diff
@@ -35,7 +35,6 @@
     "eslint": "8.56.0",
     "chai": "4.3.10",
     "mocha": "10.2.0",
-    "prettier": "3.1.0",
```
Comment (dslmeinte) [inline package.json:38]: Why is `prettier` removed?

Comment (ftomassetti) [reply package.json:38]: Really not sure, let's put it back

Comment (ftomassetti) [reply package.json:38]: oh, sorry, not, it was removed because it was a duplicate. It is already there at line 30

File: tsconfig.json (hunk 1 above)
Comment (ftomassetti) [reply tsconfig.json:17]: I am not an expert in packages, but in the tutorial I read they were using a core module directly under `src` and then optional modules under packages. It seemed a bit asymmetric to me, but I thought it was the recommended standard. If that is not the case let's move to a core package

Comment (dslmeinte) [reply tsconfig.json:17]: Not an expert either, but I'm a bit worried about the apparent lack of separation between `src/` and `packages/` re: TS-compilation. I'm sure it works as long as source code is next to each other, though.

**Comment by ftomassetti** [comment]
Now the PR got quite big, but most of the files were simply moved

**Comment by joswarmer** [comment]
Sorry, I was not notified of this PR and have done more or less the same :-(.

**Comment by joswarmer** [comment]
@ftomassetti Does your branch build correctly?

**Comment by ftomassetti** [comment]
> @ftomassetti Does your branch build correctly?

Yes, at least locally it builds. 

However, if you already re-done the same changes, I am happy to close this PR

**Comment by joswarmer** [comment]
The build problem was on the CI, as splitting into different packages invalidates a number of paths in the code.  I'll merge your PR and my branch into one and make a PR from that.

**Comment by joswarmer** [comment]
Merged changes from this branch into branch for next PR.

**Pull request closed without merging**